shared/gateway/models/application.go

```diff
@@ -2,41 +2,106 @@ package models
 
 import (
 	"context"
+
+	"github.com/pokt-foundation/portal-api-go/repository"
+	"go.mongodb.org/mongo-driver/bson/primitive"
 )
 
+type GatewayAAT struct {
```

Comment:
Long-term, all the models need to be updated to use portal-api-go repository package IMO.

Reply:
yeah but for this case the point is to migrate to the mongo equivalent which is not the same